### DISCUSSION TITLE
Multiday event (But not on following days)

### DISCUSSION THREAD

**Discussion by elzes** [alfio-event/alf.io#1436] [Q&A]
Hello,

We organize an event that's hold on 2 different Saturdays(Maybe 2 events?). We want to have tickets separate for both Saturdays an one ticket valid for both Saturdays.

So 3 type of tickets:
Event 1 (for example on 14-12-2024)
Event 2 (for example on 21-12-2024)
Event 1 +2  (Valid on 14 and 21 december)

I tried a few setups 3 separate events, one event during one week with 3 categories but with al the setup the problem stays  that the event has to last 1 whole week, while it's only 2 days. The only solution  i can think off is create 2 events with both a ticket type for both days. The problem here is off course the checkin because that is event based?

Does anyone have any thoughts about this?

**Comment by cbellone** [comment]
Hi,

alf.io is optimized for contiguous multi-day events, however support for non-contiguous multi-day events is on our roadmap. No ETA yet.

Your assumption is correct. Having attendees registered to two events (single date, multi-date) coming for the checkin will prove really problematic. I would advise against it.

I see two options here:

1. create a single event. Define ticket validity and check-in times for single-day tickets
    pros: easier to understand for attendees, easy to check-in
    cons: attendees with a two-dates ticket will receive an invitation (and a ticket) for the whole week
    
2. create two events - one for each day - and create/sell a subscription for both days
    If attendees buy a subscription, they can use the subscription code to pay for both events. You can even send tickets automatically to subscription buyer
    pros: easy check-in, two-dates attendees will receive two separate tickets
    cons: subscriptions are optimized for seasonal pass scenarios, so their use here might not be ideal

full support for non contiguous multi-day events will come in the future. 
That being said, if this is a format that you organize often, and you want to help us accelerate the development of this feature, we'd be glad to discuss it with you.

Celestino

**Comment by elzes** [comment]
Hi Celestino,

Thanks for your answer. Never tried the subscriptions but looks useful for this use case indeed (Maybe to much overhead for the customer).  .<del>I had a look at subscriptions but i miss the place where i can assign events to a subscription? Can you give me a hint? </del>

Ok found it in the edit event you can assign events to subscriptions.


Our event is always a non contiguous multi-day event, so I would love to help with this feature. Please let me know how i can help with this
